docs/mons.md

```diff
@@ -59,85 +60,126 @@ Info: Are you sure you want to restore the quorum to mon "c"? If so, enter: yes-
+Info: fetching the deployment rook-ceph-mon-b-maintenance to be running
+
+Info: deployment rook-ceph-mon-b-maintenance exists
+
+Info: Started maintenance pod, restoring the mon quorum in the maintenance pod
 Info: Extracting the monmap
 
+
 # Lengthy rocksdb output removed
 
+Info: Finished updating the monmap!
 Info: Printing final monmap
 monmaptool: monmap file /tmp/monmap
 epoch 3
-fsid 818e8153-f17f-4ae3-b21f-078a4069affa
-last_changed 2022-10-19T22:04:46.097595+0000
-created 2022-10-19T22:01:11.347220+0000
+fsid 4d32410e-fee1-4b0a-bc80-7f395fc43136
+last_changed 2024-02-27T15:42:56.748095+0000
+created 2024-02-27T15:42:03.020653+0000
 min_mon_release 17 (quincy)
 election_strategy: 1
-0: [v2:192.168.64.167:3300/0,v1:192.168.64.167:6789/0] mon.c
+0: v2:172.30.14.162:3300/0 mon.b
 Info: Restoring the mons in the rook-ceph-mon-endpoints configmap to the good mon
-configmap/rook-ceph-mon-endpoints patched
-Info: Stopping the debug pod for mon c
-setting debug mode for "rook-ceph-mon-c-debug"
-Info: removing debug mode from "rook-ceph-mon-c-debug"
-deployment.apps "rook-ceph-mon-c-debug" deleted
-deployment.apps/rook-ceph-mon-c scaled
+Info: Stopping the maintenance pod for mon b.
+
+Info: fetching the deployment rook-ceph-mon-b-maintenance to be running
+
+Info: deployment rook-ceph-mon-b-maintenance exists
+
+Info: removing maintenance mode from deployment rook-ceph-mon-b-maintenance
+
+Info: Successfully deleted maintenance deployment and restored deployment "rook-ceph-mon-b"
 Info: Check that the restored mon is responding
-timed out
-command terminated with exit code 1
-Info: 0: waiting for ceph status to confirm single mon quorum
-Info: sleeping 5
-timed out
-command terminated with exit code 1
-Info: 1: waiting for ceph status to confirm single mon quorum
-Info: sleeping 5
-timed out
-command terminated with exit code 1
-Info: 2: waiting for ceph status to confirm single mon quorum
-Info: sleeping 5
-timed out
-command terminated with exit code 1
-Info: 3: waiting for ceph status to confirm single mon quorum
-Info: sleeping 5
-
-Info: finished waiting for ceph status
-Info: Purging the bad mons: b a
-Info: purging old mon: b
-deployment.apps "rook-ceph-mon-b" deleted
-Error from server (NotFound): services "rook-ceph-mon-b" not found
-Info: purging mon pvc if exists
-Error from server (NotFound): persistentvolumeclaims "rook-ceph-mon-b" not found
-Info: purging old mon: a
-deployment.apps "rook-ceph-mon-a" deleted
-Error from server (NotFound): services "rook-ceph-mon-a" not found
-Info: purging mon pvc if exists
-Error from server (NotFound): persistentvolumeclaims "rook-ceph-mon-a" not found
-Info: Mon quorum was successfully restored to mon c
-Info: Only a single mon is currently running
-Info: Press Enter to start the operator and expand to full mon quorum again
+Error: failed to run command. failed to run command. command terminated with exit code 1%!(EXTRA string=failed to get the status of ceph cluster)
```

Comment:
Formatting error in the output

Reply:
updating [this](https://github.com/rook/kubectl-rook-ceph/pull/252/files#diff-94cb19f5706322197b23e830805b02726b6d7b75ae831c5a7ed19b1a66675a31R275) fixed the issue